python/paddle/fluid/contrib/slim/tests/test_quantization_mkldnn_pass.py

```diff
@@ -162,13 +160,18 @@ def mkldnn_based_freeze_graph(self,
                             activation_quant_type + '_' + weight_quant_type,
                             marked_nodes)
         mkldnn_program = test_graph.to_program()
-        w_mkldnn = np.array(scope.find_var('conv2d_1.w_0').get_tensor())
+
+        # Check the transformation weights of conv2d and mul
+        conv_w_mkldnn = np.array(scope.find_var('conv2d_1.w_0').get_tensor())
+        mul_w_mkldnn = np.array(scope.find_var('fc_0.w_0').get_tensor())
```

Comment:
Is there any other way of obtaining the mul and conv weights rather than by searching for their names directly?